docs/Architecture.md

```diff
@@ -3,6 +3,30 @@ id: architecture
 title: Architecture
 ---
 
+This diagram shows how Jest works to run your tests. Jest is composed of a bunch of internal modules, however, to keep things a bit easier, we've decided to focus at a higher level.
+
+![](https://i.imgur.com/qGVNhBB.png)
+
+**Jest-Cli**: Module responsible for exposing the communication interface to the users.
```

Comment:
instead of `**Jest-Cli**`, could we use `` `jest-cli` ``? same for the others - use the actual package name in backticks